### ISSUE TITLE
Shipping center POI is marked at wrong place on CAF Goose Bay Invasion v1 in squadlanes.

### ISSUE THREAD

**Issue by TheoryO** [w4rum/squadlanes#21]
Just used squadlanes on EGC server and the last point of Canadian(defender/Team1), the Shipping center was wrongly marked in the squadlanes. Compare CAF Invasion v1 layer with CAF RAAS v1 layer and you'll see what I mean.

Always thank you for the site! It is by far greatest squad tool in the web.

**Comment by w4rum** [comment]
I see what you mean. I'll get it out.

For reference:
![image](https://user-images.githubusercontent.com/1405498/140535296-93fe5fb0-b9a7-442a-9e17-43c324ab3450.png)
![image](https://user-images.githubusercontent.com/1405498/140535324-d2f38159-156f-4a43-93e2-774b5ba145cb.png)

**Comment by w4rum** [comment]
Hmm, re-extracting the game files did not fix the issue.

You are saying that you were playing `Goose Bay CAF Invasion v1` and `Shipping Center` was to the west of the main base (as shown on image 2) and not to the north of the main base (as on image 1), correct?

If so, which game version are you running?

**Comment by TheoryO** [comment]
Sorry about late reply...
Yes, Shipping center popped on west of main base instead of north.
I can't check version but the incident happened on the day I posted the
github thread, so I'm pretty sure it's latest version.

2021년 11월 6일 (토) 오전 1:31, Tim Schmidt ***@***.***>님이 작성:

**Comment by w4rum** [comment]
Okay, then I can't explain this. I'm extracting the capture point locations from the same map files that the game is using. And I don't see this issue anywhere else so I doubt it's a structural problem with my extraction scripts.

I'll make another extraction run soon. Maybe my Squad files were out of date for some reason. I don't have a lot of time that I can dedicate to this at the moment, so I can't promise to fix this in the near future.

**Comment by TheoryO** [comment]
I just loaded the layer in training and there it shows Shipping center west of main base
![2021-11-18 (1)](https://user-images.githubusercontent.com/93781080/142391159-47ea1b64-29a2-40a3-b867-bd54f2a8d546.png)

![2021-11-18](https://user-images.githubusercontent.com/93781080/142391171-f75037f3-fdde-42bb-bbb6-40a2c1381d55.png)

**Comment by w4rum** [comment]
Thanks a lot for the screenshots, including the exact version tag! I'll take a look at what's wrong with my extraction setup once I find the time.

**Comment by w4rum** [comment]
Fixed in 762687980bd3067e636c32c647fc85d7ca69869e.

Sorry for the very long turnaround time. Finally had the time and motivation to look into this. Turns out the cluster containing `Shipping Center` and `Happy Valley` has a rotation, which makes it the *only* cluster in the entire game to have a rotation. I assumed I didn't have to care about rotation because it was always 0 in every cluster I examined initially. But, hey, rotation is now supported.
